bindings/go/ctf/ctf_compatibility_test.go

```diff
@@ -45,7 +45,7 @@ func Test_CTF_Basic_ReadOnly_Compatibility(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			ctx := t.Context()
 			r := require.New(t)
-			archive, discovered, err := ctf.OpenCTFByFileExtension(ctx, tc.path, ctf.O_RDONLY)
+			archive, discovered, err := ctf.OpenCTFByFileExtension(ctx, tc.path, ctf.O_RDONLY, nil)
```

Comment:
am I the only one that hates nil parameters? maybe :D

Reply:
Nah, for sure. But this is the default I guess. That's why I added an extra test to test the config explicitly. What would you prefer? An options list?